sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -121,7 +121,7 @@ export class StudentController extends BaseController {
         phone: existingStudent.contactInfo.phone,
       },
       pdVerified: existingStudent.studentPDVerified,
-      validSin: existingStudent.sinValidation.isValidSIN,
+      validSin: existingStudent.sinValidation?.isValidSIN,
```

Comment:
What is the scenario when _sinValidation_ is null?

Reply:
You are right, I added it as part of the blue box testing. Updated now.